tensorflow/python/ops/array_ops.py

```diff
@@ -1331,6 +1331,8 @@ def parallel_stack(values, name="parallel_stack"):
     output: A stacked `Tensor` with the same type as `values`.
   """
   with ops.name_scope(name):
+    if context.executing_eagerly():
+      return stack(values, name=name)
```

Comment:
Seems like we are stating that in eager mode parallel_stack doesn't really offer any benefits over stack since all the inputs have been computed. If so, I think we should update the documentation to state that parellel_stack is the same as stack in eager mode and thus does not have caveats around backprop.

WYDT @alextp ?

Reply:
I think just silently using stack in eager mode is fine.

Reply:
Cool, I think just updating the documentation a bit would be great.